Clicker/Views/GroupControls/GroupControlsInfoCell.swift

```diff
@@ -1,100 +1,99 @@
 //
-//  GroupControlsInfoView.swift
+//  InfoViewCell.swift
 //  Clicker
 //
-//  Created by Kevin Chan on 11/10/18.
-//  Copyright © 2018 CornellAppDev. All rights reserved.
+//  Created by Mathew Scullin on 3/10/19.
+//  Copyright © 2019 CornellAppDev. All rights reserved.
 //
 
 import UIKit
+import SnapKit
 
-class GroupControlsInfoView: UIView {
-
+class GroupControlsInfoCell: UICollectionViewCell {
+    
     // MARK: - View vars
     var numMembersLabel: UILabel!
     var numPollsLabel: UILabel!
     var codeLabel: UILabel!
     var separatorLineView1: UIView!
     var separatorLineView2: UIView!
-
+    
     // MARK: - Constants
     let horizontalPadding: CGFloat = 15
     let separatorLineWidth: CGFloat = 1.5
-
+    
     override init(frame: CGRect) {
         super.init(frame: frame)
-
+        
```

Comment:
<img width="803" alt="Screen Shot 2019-03-14 at 2 37 13 AM" src="https://user-images.githubusercontent.com/26048121/54336307-1b18cd80-4602-11e9-96c7-1f5ebc4be9a3.png">
make sure you have the trim whitespaces option checked in your Xcode preferences